docs/learn/storage/block-sync.md

```diff
@@ -0,0 +1,73 @@
+#  Block sync
+
+Block sync is a process of following up the latest blocks and states. Node operators can choose from various methods that best fits the machine spec and service requirements.
+
+## Full sync
+
+Full sync generates the block states by downloading every block (header and transactions) and executing it. The blocks are downloaded by p2p peers. A full sync is activated with `--syncmode full` flag or simplly omitting the flag, because full sync is the default sync method.
```

Comment:
```suggestion
Full sync generates the block states by downloading every block (header and transactions) from p2p peers and executing it. A full sync is activated with `--syncmode full` flag or simply omitting the flag, because full sync is the default sync method.
```